pkg/lightning/config/config.go

```diff
@@ -100,6 +103,7 @@ const (
 
 	defaultCSVDataCharacterSet       = "binary"
 	defaultCSVDataInvalidCharReplace = utf8.RuneError
+	defaultRemoteBackendChunkSize    = 8 * 1024 * 104
```

Comment:
```suggestion
	defaultRemoteBackendChunkSize    = 8 * 1024 * 1024
```

typo? Or you can use units.MiB like line 100